src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,109 @@
+package pages;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class HomePage extends BasePage {
+
+    WebDriver driver;
+    WebDriverWait wait;
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )


![image](https://user-images.githubusercontent.com/104761482/214282820-fe493007-61c2-4994-8a0a-df21b9bce8a7.png)

![image](https://user-images.githubusercontent.com/104761482/214282929-442ebf21-5fb7-4efe-bf89-ed39796ff32e.png)